src/app/query/Order.query.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import { Field } from 'Util/Query';
+import { OrderProductQuery } from 'Query';
+
+/**
+ * Order Query
+ * @class OrderQuery
+ */
+export class OrderQuery {
+    getOrderListQuery() {
```

Comment:
Please redo completely by referencing our queries. If you notice, they never reference a addField, but addFieldList. Otherwise it is too hard to extend!!!